### DISCUSSION TITLE
Explantation of entities

### DISCUSSION THREAD

**Discussion by bifbaf** [sfstar/hass-victron#135] [General]
Plugin works great for me but I have some difficulties to choose the right values for my needs.

Can anyone explain to me which value I have to use for the home assistant dashboard to get the total amount of PV energie in kWh (solar production)?

Also it is not fully clear to me which entities I should use for the batterie.

**Comment by skorunka** [comment]
Yes, the documentation of all the sensors provided would be very helpful.

**Comment by attimcz** [comment]
I did a quick mapping of hass-victron entities to Modbus registers. The mapping is based on the const.py file in hass-victron v0.1.7 and Victron Modbus register list v. 3.20.
Maybe this will help someone
[hass-victron_entity_to_modbus_mapping.xlsx](https://github.com/sfstar/hass-victron/files/14640433/hass-victron_entity_to_modbus_mapping.xlsx)
